spec/system/jobseekers_can_submit_a_job_application_spec.rb

```diff
@@ -13,18 +15,23 @@
   context "when the application is complete" do
     let(:job_application) { create(:job_application, :complete, jobseeker: jobseeker, vacancy: vacancy) }
 
-    it "allows jobseekers to submit application and go to confirmation page" do
+    it "allows jobseekers to submit application and receive confirmation email" do
       visit jobseekers_job_application_review_path(job_application)
 
       click_on I18n.t("buttons.submit_application")
       expect(page).to have_content("There is a problem")
 
       check "Confirm data accurate"
       check "Confirm data usage"
-
-      expect { click_on I18n.t("buttons.submit_application") }
+      expect { perform_enqueued_jobs { click_on I18n.t("buttons.submit_application") } }
         .to change { JobApplication.first.status }.from("draft").to("submitted")
+        .and change { delivered_emails.count }.by(1)
+
       expect(page).to have_content(I18n.t("jobseekers.job_applications.submit.panel.title"))
+
+      # TODO: expect that the first link goes to the view your application path, once that page has been created
```

Comment:
This page has been created now

Reply:
But it's easier to write placeholder code than real code!